test/addons-napi/test_properties/test.js

```diff
@@ -39,3 +39,7 @@ test_object.readwriteAccessor2 = 2;
 assert.strictEqual(test_object.readwriteAccessor2, 2);
 assert.strictEqual(test_object.readonlyAccessor2, 2);
 assert.throws(() => { test_object.readonlyAccessor2 = 3; }, TypeError);
+
+assert.ok(test_object.hasNamedProperty(test_object, 'echo'));
```

Comment:
It might be better to use `strictEqual()` here to make sure we're actually getting a Boolean back.